From: Richard B Winters <rik@mmogp.com>
Date: Tue, 5 May 2015 20:42:01 -0400
Subject: Fix pcsd service file for Debian

 - Updates environment configuration file location
 - Updates ExecStart
 - Removes conflicting options

Change-Id: I01b7a147be2c63ff0f876666c47eb8fb01b193c4
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 pcsd/pcsd.service | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/pcsd/pcsd.service b/pcsd/pcsd.service
index 5254905..36477de 100644
--- a/pcsd/pcsd.service
+++ b/pcsd/pcsd.service
@@ -2,11 +2,8 @@
 Description=PCS GUI and remote configuration interface
 
 [Service]
-EnvironmentFile=-/etc/sysconfig/pcsd
-WorkingDirectory=/var/lib/pcsd
-ExecStart=/usr/lib/pcsd/pcsd start
-Type=notify
-NotifyAccess=all
+EnvironmentFile=/etc/default/pcsd.conf
+ExecStart=/usr/bin/ruby -C/var/lib/pcsd -I/usr/share/pcsd -- /usr/share/pcsd/ssl.rb & > /dev/null &
 
 [Install]
 WantedBy=multi-user.target
